### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by maoschanz** [maoschanz/drawing#157]
## Description

Pressing tab works in the headerbar but:

- on tools, it focuses each unselectable flowboxchild before the actual button
- it losts entirely the focus after the last tool (maybe it's focusing hidden panels and gets stuck?)
- might be stuck on the GtkLayout? Happens if pressing "right arrow" while a tool is selected

----

## System

- Device (desktop computer/tablet/phone/...): desktop computer
- App version: 0.5.2

**Comment by maoschanz** [comment]
>it losts entirely the focus after the last tool (maybe it's focusing hidden panels and gets stuck?)

maybe not

```
(com.github.maoschanz.drawing:3): Gtk-CRITICAL **: 20:10:19.915: gtk_widget_event: assertion 'WIDGET_REALIZED_FOR_EVENT (widget, event)' failed
```
